sae_lens/loading/pretrained_sae_loaders.py

```diff
@@ -1326,6 +1326,66 @@ def mwhanna_transcoder_huggingface_loader(
     return cfg_dict, state_dict, None
 
 
+def mntss_clt_layer_huggingface_loader(
+    repo_id: str,
+    folder_name: str,
```

Comment:
Nit: Call this "layer_num" and make it an int since that's a little more descriptive.

Reply:
Ah thanks! This is fine.